### PR TITLE
Modeling

### PR DESCRIPTION
**Pull request by Coriana1**

### CODE REVIEW

**Comment by Coriana1** [comment]
- [x] Top-level README "Table of Contents" is updated
 - [x] README for this challenge is complete
       - [x] Summary, Description, Approach & Efficiency, Solution
       - [x] Picture of whiteboard
       - [x] Link to code
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
       - [n/a] "Happy Path" - Expected outcome
       - [n/a] Expected failure
       - [n/a] Edge Case (if applicable/obvious)

File: src/routes/food.js [hunk 1]
```diff
@@ -54,7 +52,7 @@ router.put('/food/:id', async (req, res) => {
 
 router.delete('/food/:id', async (req, res) => {
   try {
-    const deletedRowsCount = await Food.destroy({ where: { id: req.params.id } });
+    const deletedRowsCount = await food.destroy({ where: { id: req.params.id } });
```
Comment (lizkavalski) [inline src/routes/food.js:55]: Replace destroy with delete because the file is referencing the collection file to run the function and replace `{ where: { id: req.params.id } })` with just `req.params.id`. Then remove the if/else statement and replace it with a 200 status and a send/json for example `res.status(200).send({ message:'has been deleted', deletedRowsCount})`

**Review by lizkavalski — COMMENTED** [review]
You are close. Frist focus one getting CURD to work in both models (food and clothes), and then association part.

File: src/routes/food.js [hunk 2]
```diff
@@ -1,30 +1,28 @@
...
-router.get('/food', async (req, res) => {
-  try {
-    const foods = await Food.findAll();
-    res.status(200).send(foods);
-  } catch (error) {
-    res.status(500).send({ error: 'Failed to fetch the records' });//change the rest to .send
-  }
+// get all records
+router.get('/food', async (req, res, next) => {
+  let foods = await food.findAll();
```
Comment (lizkavalski) [inline src/routes/food.js:19]: Replace findAll with find
